src/cocoa/toga_cocoa/widgets/base.py

```diff
@@ -11,6 +11,10 @@ def __init__(self, interface):
         self.native = None
         self.create()
         self.interface.style.reapply()
+        self.set_enabled(self.interface.enabled)
```

Comment:
Are there widgets that aren't enabled by default?

Reply:
No, but if you currently pass `enabled=False` to the constructor you still end up with an enabled widget and not  disabled one.